koboldcpp.py

```diff
@@ -804,8 +804,45 @@ def string_contains_sequence_substring(inputstr,sequences):
 currfinishreason = "null"
 using_gui_launcher = False
 using_outdated_flags = False
+using_openai_tools = False
```

Comment:
I think rather than using a global flag, we should use a local flag within the generate call instead. I'll do a bit of refactoring later